app/src/main/java/com/android/sample/ui/authentication/SignInScreen.kt

```diff
@@ -188,7 +190,10 @@ fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
           // Show the loading animation if the user is registered
           if (Firebase.auth.currentUser != null || registered.value) {
             LoadingAnimation(
-                onFinish = { navigationActions.navigateTo(Screen.SWIPE) },
+                onFinish = {
+                  userViewModel.getCurrentUser(isConnected)
```

Comment:
Nice catch ! I think this also fix the issue #270. Is it possible to notify @CapraSnitzel ?

Reply:
Yes, I just want to discuss it further with him, as there are some points I need clarification on. For example, why we need to call `getCurrentUser` each time we navigate to the user account screen. I’ll bring this up during the next meeting tomorrow and get back to you afterward. Thank you for your patience!